package.json

```diff
@@ -1,21 +1,20 @@
 {
   "name": "node-mvm",
-  "version": "1.3.0",
+  "version": "1.5.0-NN",
   "description": "Mainscreen View Manager - Mod for Artemis Space Ship Bridge Simulator - Control what is shown on your Main Screen clients! Uses a TCP proxy that injects changes into Artemis packets.",
   "main": "mvm.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/UserMcUser/node-MVM"
+    "url": "https://github.com/NoseyNick/node-MVM"
```

Comment:
Oh this you are obviously more than welcome to edit back to **your** URL of course!   :laughing: